frontend/src/lib/components/Orgs/OrgMemberTable.svelte

```diff
@@ -0,0 +1,135 @@
+<script lang="ts">
+  import { Button } from '$lib/forms';
+  import t from '$lib/i18n';
+  import { AdminIcon, Icon } from '$lib/icons';
+  import { createEventDispatcher } from 'svelte';
+  import Dropdown from '../Dropdown.svelte';
+  import type { User } from '../../../routes/(authenticated)/admin/+page';
+  import type { OrgRole } from '$lib/gql/types';
+  import OrgRoleSelect from '$lib/forms/OrgRoleSelect.svelte';
+  import OrgRoleText from './OrgRoleText.svelte';
+  import AddOrgMember from '../../../routes/(authenticated)/org/[org_id]/AddOrgMember.svelte';
```

Comment:
it feels like we should't be importing a component in a route from lib. Should we either move the org table component into the route or the add org member component into lib?